Adding IsConnected() convenience function to session

abramovic opened the pull request.

abramovic committed:
Adding IsConnected() convenience function to session

omeid commented on session.go:
```diff
@@ -120,6 +120,17 @@ func (o *CloseOpts) toMap() map[string]interface{} {
 
+// Convenience function that says whether client is still connected
+func (s *Session) IsConnected() bool {
+	if s.closed == true {
```
Or simply `return !s.closed && s.cluster != nil`.

dancannon commented:
Hey thanks for the PR, looks great so far. I agree with the comments that @alkchr made, if possible could you look into making the suggested changes?

abramovic committed:
Added changes to IsConnected()

- Made change in comment style based off of the rest of the comments in library
- Merged two if statements into one (session.cluster==nil and session.closed)
- Still returns the IsConnected value for the cluster

abramovic committed:
go fmt

abramovic commented:
Hey guys thanks for the feedback. 
- Made change in comment style based off of the rest of the comments in library
- Merged the two if statements into one (session.cluster==nil and session.closed)
- Will return value of session.cluster.IsConnected() if session.cluster is not nil

dancannon commented:
Awesome :+1: Will merge when I release v1.3.0. Thanks again for the contribution.
